devsetup/Makefile

```diff
@@ -273,12 +276,26 @@ bmaas_crc_baremetal_bridge_cleanup:
 
 .PHONY: bmaas_baremetal_net_nad
 bmaas_baremetal_net_nad:
+	pushd .. && make namespace && popd
 	scripts/bmaas/network-attachement-definition.sh --create
 
 .PHONY: bmaas_baremetal_net_nad_cleanup
 bmaas_baremetal_net_nad_cleanup:
 	scripts/bmaas/network-attachement-definition.sh --cleanup
 
+.PHONY: bmaas_metallb
+bmaas_metallb: export NETWORK_NAME = ${BMAAS_NETWORK_NAME}
+bmaas_metallb: export POOL_NAME = ${BMAAS_METALLB_POOL_NAME}
+bmaas_metallb: export ADDRESS_POOL = ${BMAAS_METALLB_ADDRESS_POOL}
+bmaas_metallb:
+	pushd .. && make metallb && popd
```

Comment:
Sorry I'm  a little late to review this. But I think we should avoid these cross Makefile target dependencies.

Reply:
I don't mind removing them, and instead document that you need to do `make metallb` manually as a pre-requisite to using the `bmaas_metallb` target. We can add a check to the script and fail `bmaas_metallb` target with a good error message instead. I went this way, because we already did it this way for `input`/`namespace` and `nmstate` targets.